test/mri/excludes/DRbTests/TestDRbCore.rb

```diff
@@ -0,0 +1,3 @@
+exclude :test_02_unknown, "<\"DRbTests::DRbEx::\"> expected but was <\"DRbTests::DRbEx::FooBar\">."
+exclude :test_06_timeout, "[Timeout::Error] exception expected, not #<DRb::DRbConnError: execution expired>."
+exclude :test_11_remote_no_method_error, "[DRb::DRbRemoteError] exception expected, not #<NoMethodError: undefined method `invoke_no_method' for #<DRbTests::DRbEx:0x4d8>>."
```

Comment:
That one at least it's from `test/mri/failing.exclude` which excludes test/mri/tests/drb/test_drb.rb entirely.

Maybe we should remove the drb tests from there then if they work?
One concern about DRb is DRb uses ObjectSpace._id2ref which can be really slow, so tests doing that should probably be tagged as slow/very slow, which might happen automatically.

Reply:
Not sure about TestBundlerGem I don't anything excluding that. You could check in CI logs to see if that gets run in CI currently. If yes, then maybe there are extra failures when run by @nirvdrum compared to CI for some reason?

Reply:
I ran a `for` loop in my shell over each of the MRI test files and ran `jt retag` on them. It looks like we originally tagged these files differently. I think it's quite convenient being able to retag everything in a shell loop, so I'd like to keep the new tags. I didn't know about _test/mri/failing.exclude_.

I can either delete the entries there and keep the individual tags or I can update `jt retag` to compare the current test file against whatever's in _failing.exclude_. I think I'm slightly partial to the former since that would give us reasons for each exclusion. But, if we just don't support a particular feature, I could see a bulk block making sense.

Reply:
I'm surprised `jt test mri FILE` when FILE is in `failing.exclude` would even run given on master it doesn't in that case:
```
$ jt test mri test/mri/tests/drb/test_drb.rb
Using Interpreted TruffleRuby: mxbuild/truffleruby-jvm
No test files! (probably filtered out by failing.exclude)
```
But I guess with this PR changes it behaves/behaved differently.

Anyway, if these can run it seems fine to remove them from `failing.exclude`, unless marked as `too slow`.
It's also OK to do nothing about these and just merge as it is, as it might be a rabbit hole to remove from this failing list.